### PR TITLE
#5096 Add tests to verify settings of workspace works normally

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5096)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Topic/TopicCreateDialog.js [hunk 1]
```diff
@@ -79,6 +79,7 @@ const TopicCreateDialog = ({ broker, isOpen, onClose, onConfirm, topics }) => {
               <Field
                 type="text"
                 label="Topic name"
+                id="Topic name"
```
Comment (eechih) [inline ohara-manager/client/src/components/Topic/TopicCreateDialog.js:82]: What is the purpose of adding the **id** attribute?

Comment (saivirtue) [reply ohara-manager/client/src/components/Topic/TopicCreateDialog.js:82]: from [meterial ui doc](https://material-ui.com/api/text-field/)
> The id of the input element. Use this prop to make label and helperText accessible for screen readers.

Comment (eechih) [reply ohara-manager/client/src/components/Topic/TopicCreateDialog.js:82]: I get it :)

**Comment by saivirtue** [comment]
retry manager-e2e

**Pull request merged**